src/sentry/integrations/vercel/client.py

```diff
@@ -42,6 +46,9 @@ def get_projects(self):
         # TODO: we will need pagination since we are limited to 20
         return self.get(self.PROJECTS_URL)["projects"]
 
+    def get_source_code_provider(self, vercel_project_id):
+        return self.get(self.PROJECT_URL % vercel_project_id)["link"]["type"]
```

Comment:
@ceorourke Is `link` in the docs? I'm a bit concerned about using an undocumented API field so we might need to contact them.

Reply:
It is not, here is the page: https://vercel.com/docs/api#endpoints/projects/get-a-single-project 
but I'm not seeing any other endpoint that'd give us information about which source code tool they've connected the project to, will keep looking.

Reply:
@scefali  It is not, here is the page: https://vercel.com/docs/api#endpoints/projects/get-a-single-project
but I'm not seeing any other endpoint that'd give us information about which source code tool they've connected the project to, will keep looking.